### DISCUSSION TITLE
【投票】对于作品命名后缀的投票

### DISCUSSION THREAD

**Discussion by Johnserf-Seed** [Johnserf-Seed/TikTokDownload#588] [Polls]
新的命名规则采用模板方式来进行，分隔仅支持下划线“_”与横线“-”
```
{nickname}：用户名
{create_time}：发布时间
{aid}：作品id
{desc}：作品文案
```
本次投票的内容是关于是否要给不同的文件类添加后缀，例如
```
第一种
2022-01-31 22-05-21_示例_image_1.jpg
2023-08-02 11-03-55_示例_video.mp4
```
和
```
第二种
2022-01-31 22-05-21_示例1.jpg
2023-08-02 11-03-55_示例.mp4
```

<hr>

The new naming rules are done in a template fashion, and delimitation is only supported for underscores "_" and crosses "-"
```
{nickname}: username
{create_time}: publish time
{aid}: id of the work
{desc}: text of the work
```
This poll is about whether to add suffixes to different file classes, for example
```
The first one
2022-01-31 22-05-21_example_image_1.jpg
2023-08-02 11-03-55_example_video.mp4
```
and
```
The second one
2022-01-31 22-05-21_example_1.jpg
2023-08-02 11-03-55_example.mp4
```

**Comment by JoeanAmier** [comment]
`mp4`和`jpg`后缀足以区分类型。

**Comment by leviettung01** [comment]
此时无法加载照片。这是一个错误吗？

**Reply by zpq5935** [reply]
我希望支持文件名取部分，有的desc太长，只能全用的话好像有些文件系统并不支持

**Reply by Johnserf-Seed** [reply]
新版解决了这个过长的问题，过长自动单独保存文案

**Reply by zpq5935** [reply]
太好了，是哪个版本来着
那对应视频或者文件夹那些的名称是怎样的规则来着

**Reply by zpq5935** [reply]
不知道这个规则是否会影响到文件检查，导致已存在文件重新下载呢

**Reply by Johnserf-Seed** [reply]
在f2工具发布后我就会把TikTokDownload同步更新。规则都是全局的更新的，如果修改了就会重新下载。如果只对未下载的作品应用新规则则会带来新问题，旧的文件夹和视频不会自动合并，所以暂时的考虑是更新规则重新下载。但是新版多了数据库所以会记录下载的作品与路径绑定。

**Reply by leviettung01** [reply]
当我上传照片时出现此错误：该链接可能无法访问。 异常：HTTP连接意外: 403
下载视频还可以正常，但是下载照片就无法下载了。可以修复吗？

**Comment by wangnan789** [comment]
希望能用下划线_替换空格

**Reply by Johnserf-Seed** [reply]
空格默认代替为下划线的
